flexget/plugins/search_kat.py

```diff
@@ -69,34 +78,66 @@ def search(self, task, entry, config):
                 elif r.status_code != 200:
                     log.warning('Search returned %s response code' % r.status_code)
                     continue
-                rss = feedparser.parse(r.content)
 
-                ex = rss.get('bozo_exception', False)
-                if ex:
-                    log.warning('Got bozo_exception (bad feed)')
-                    continue
+                entries |= self.parse_rss(r.content) if config.get('rss') else self.parse_web(r.content)
 
-                for item in rss.entries:
-                    entry = Entry()
-                    entry['title'] = item.title
+        return entries
+
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/kristen/Flexget/file/2167275111/issues/source?bid=3161667&fileBranchId=3161675#l86)